IpmiFeaturePkg/Include/Library/PlatformPowerRestorePolicyConfigurationLib.h

```diff
@@ -0,0 +1,34 @@
+/** @file
+
+    PlatformPowerRestorePolicyConfigurationLib.h
+
+    Copyright (c) Microsoft Corporation.
+**/
+
+#ifndef PLATFORM_POWER_RESTORE_POLICY_CONFIGURATION_LIB_H_
+#define PLATFORM_POWER_RESTORE_POLICY_CONFIGURATION_LIB_H_
+
+//
+// Power Restore Policy configuration.

```

Comment:
Maybe make it clear that these definitions come from the IPMI spec.

Reply:
updated in https://github.com/microsoft/mu_feature_ipmi/pull/144/commits/4c48d83110b5669392c1f3a70926dc1c133d3cd6